.evergreen/config.in.yml

```diff
@@ -206,47 +184,21 @@ functions:
             export AWS_REGION='${AWS_REGION}'
             export AWS_CMK_ID='${AWS_CMK_ID}'
             export AWS_DEFAULT_REGION='us-east-1'
-            export KMIP_TLS_CA_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
-            export KMIP_TLS_CERT_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem"
+            export CSFLE_TLS_CA_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
+            export CSFLE_TLS_CLIENT_CERT_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem"
           EOT
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
-        working_dir: src
         timeout_secs: 300
-        shell: bash
-        script: |
-          ${PREPARE_SHELL}
-
-          # Disable xtrace (just in case it was accidentally set).
-          set +x
-          source ./prepare_client_encryption.sh
-          rm -f ./prepare_client_encryption.sh
-
-          export VERSION=${VERSION}
-          export DRIVERS_TOOLS=${DRIVERS_TOOLS}
-
-          source ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
-
-          echo "CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
-
-          # Get access to the AWS temporary credentials:
-          echo "adding temporary AWS credentials to environment"
-          # CSFLE_AWS_TEMP_ACCESS_KEY_ID, CSFLE_AWS_TEMP_SECRET_ACCESS_KEY, CSFLE_AWS_TEMP_SESSION_TOKEN
-          pushd "$DRIVERS_TOOLS"/.evergreen/csfle
-          . ./activate-kmstlsvenv.sh
-          . ./set-temp-creds.sh
-          popd
-
-          export MONGODB_API_VERSION="${MONGODB_API_VERSION}"
-          export AUTH="auth"
-          export SSL="ssl"
-          export TEST_CSFLE=true
-
-          source secrets-export.sh
-          source serverless.env
-
-          bash ${PROJECT_DIRECTORY}/.evergreen/run-serverless-tests.sh
+        working_dir: src
+        binary: bash
+        env: 
+          AUTH: 'auth'
+          SSL: 'ssl'
+        add_expansions_to_env: true
+        args: 
+          - .evergreen/run-serverless-tests.sh
```

Comment:
Half of this block was configuring FLE variables, the other half re-exported expansions.  It can all be removed.